sale_order_line_cancel/__manifest__.py

```diff
@@ -0,0 +1,20 @@
+# © 2016 Sylvain Van Hoof
+# Copyright 2023 ACSONE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    "name": "Sale Order Line Cancel",
+    "version": "16.0.1.0.0",
+    "author": "Camptocamp, ACSONE SA/NV, Odoo Community Association (OCA)",
```

Comment:
Let's keep original authors
```suggestion
    "author": "Okia, BCIM, Camptocamp, ACSONE SA/NV, Odoo Community Association (OCA)",
```